bitrise.yml

```diff
@@ -367,13 +367,14 @@ workflows:
             set -e
             # debug log
             set -x
-            ./screenshots.sh --test-without-building $MOZ_LOCALES
+            ./l10n-screenshots.sh --test-without-building $MOZ_LOCALES
```

Comment:
Naming the scripts as on firefox-ios to be sure the environment is the same, once it works we can rename/modify